### DISCUSSION TITLE
License?

### DISCUSSION THREAD

**Discussion by tropChaud** [LOLAPPS-Project/LOLAPPS#24] [Q&A] [answered]
Hey all, thanks for this fantastic resource. Is there any intention to add a license similar to the other LOL projects? Any restrictions on reuse of the repo content?

**Comment by redr0nin** [comment, marked as answer]
No restrictions. Just implemented GPL-3.0, as other LOL projects have done.

**Reply by tropChaud** [reply]
Amazing - thanks!
